_installation/single-multiple.md

```diff
@@ -26,16 +33,59 @@ CREATE USER babelfish_user WITH CREATEDB
 	CREATEROLE PASSWORD '<PUT_SECRET_PASS_HERE>' INHERIT;
 DROP DATABASE IF EXISTS demo;
 
+/* the Babelfish database */
 CREATE DATABASE demo OWNER babelfish_user;
+ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
+SELECT pg_reload_conf();
+
+/* determine the migration mode */
+ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 
 \c demo
 
 CREATE EXTENSION IF NOT EXISTS "babelfishpg_tds" CASCADE;
-ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
-ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 CALL SYS.INITIALIZE_BABELFISH('babelfish_user');
 ```
 
-The `ALTER DATABASE` statement assigns the correct parameter to the database.
-The init function will then perform all the steps needed to perform the setup.
+The `ALTER DATABASE` statement assigns the correct migration mode to the
+database.  The `SYS.INITIALIZE_BABELFISH` function will then perform all
+the steps needed to set up Babelfish.
```

Comment:
You need to move this part to a separate section named 'Provisionign the babelfish DB' or something like that.  The fundamental issue here is that the migration_mode is just an input into the provisioning step, and it makes no difference for how that step goes. The current test looks like the migration mode is a defining choice for the provisioning and that is just not true. Provisioning is a system-level event, where the migation mode is a user-facing functionality aspect. So separate these two from a doc perspective.

Reply:
Please clarify that.
Are you saying that `migration_mode` is irrelevant apart from how it is set during the call to `SYS.INITIALIZE_BABELFISH`?
If yes, why don't we just set it with `SET` before calling the procedure?